### ISSUE TITLE
about the screensaver pattern

### ISSUE THREAD

**Issue by saltat** [BOINC/boinc#5979]
### Describe the bug

I have let it run only in the night, but I do not want to open it everyday, so I did not quit it during the daytime. I do not want it to show its screen saving patterns when I'm not using my computer, because it is really annoying to see a screen saving started when I'm reading or watching something on my computer. How can I stop it?

**Comment by AenBleidd** [comment]
I don't believe there is any OS that let you set different screensaver settings for day and night.
BOINC just uses OS built-in functionality to show screensaver, so there is nothing customized present.
I suggest to ask this question to your OS provider.
